sdk/identity/azure-identity/src/client_secret_credential.cpp

```diff
@@ -36,20 +36,20 @@ std::string UrlEncode(std::string const& s)
 }
 } // namespace
 
-std::string const ClientSecretCredential::g_aadGlobalAuthority
+std::string const TokenCredentialOptions::g_aadGlobalAuthority
```

Comment:
should this be in ::details?

Reply:
Does it need to be global or can it be a static member in the TokenCredentialOptions class?

Reply:
static is global, but with class/struct scope, so syntax is a git different. It is private static, so it would not be a breaking change to move it to Details.

Reply:
I was referring to the g_aad* name.  Our guidelines are g_ is for globals.  This is not a global so it shouldn't be g_

Reply:
Statics are globals, we had that discussion. Prefixes for statics are exactly the same, `g_`.